antarest/study/storage/rawstudy/raw_study_service.py

```diff
@@ -130,24 +127,11 @@ def create(self, metadata: RawStudy) -> RawStudy:
         Returns: new study information
 
         """
-        version_template: Optional[str] = STUDY_REFERENCE_TEMPLATES.get(
-            metadata.version, None
+        create_new_empty_study(
```

Comment:
just use this method to extract the study to the correct path, then do the rest concerning the RawStudy here.